@staticfloat, I'm seeing an odd clang failure on MacOS Travis that I can't replicate on my local machine (with `Apple LLVM version 10.0.1 (clang-1001.0.46.4)`).  Any ideas?  It used to build fine, so I'm not sure what changed. 
```
_fftw_rdft_conf_standard in librdft.a(conf.o)
  "_fftw_have_simd_avx2_128", referenced from:
      _fftw_dft_conf_standard in libdft.a(conf.o)
      _fftw_rdft_conf_standard in librdft.a(conf.o)
  "_fftw_have_simd_sse2", referenced from:
      _fftw_dft_conf_standard in libdft.a(conf.o)
      _fftw_rdft_conf_standard in librdft.a(conf.o)
  "_fftw_join_taint", referenced from:
      _fftw_mkproblem_dft in libdft.a(problem.o)
      _fftw_mkproblem_rdft in librdft.a(problem.o)
      _fftw_mkproblem_rdft2 in librdft.a(problem2.o)
  "_fftw_rdft_hb_genus", referenced from:
      _desc in librdft_scalar_r2cb.a(hb_2.o)
      _desc in librdft_scalar_r2cb.a(hb_3.o)
      _desc in librdft_scalar_r2cb.a(hb_4.o)
      _desc in librdft_scalar_r2cb.a(hb_5.o)
      _desc in librdft_scalar_r2cb.a(hb_6.o)
      _desc in librdft_scalar_r2cb.a(hb_7.o)
      _desc in librdft_scalar_r2cb.a(hb_8.o)
      ...
  "_fftw_rdft_hc2cb_genus", referenced from:
      _desc in librdft_scalar_r2cb.a(hc2cb_2.o)
      _desc in librdft_scalar_r2cb.a(hc2cb_4.o)
      _desc in librdft_scalar_r2cb.a(hc2cb_6.o)
      _desc in librdft_scalar_r2cb.a(hc2cb_8.o)
      _desc in librdft_scalar_r2cb.a(hc2cb_10.o)
      _desc in librdft_scalar_r2cb.a(hc2cb_12.o)
      _desc in librdft_scalar_r2cb.a(hc2cb_16.o)
      ...
  "_fftw_rdft_hc2cf_genus", referenced from:
      _desc in librdft_scalar_r2cf.a(hc2cf_2.o)
      _desc in librdft_scalar_r2cf.a(hc2cf_4.o)
      _desc in librdft_scalar_r2cf.a(hc2cf_6.o)
      _desc in librdft_scalar_r2cf.a(hc2cf_8.o)
      _desc in librdft_scalar_r2cf.a(hc2cf_10.o)
      _desc in librdft_scalar_r2cf.a(hc2cf_12.o)
      _desc in librdft_scalar_r2cf.a(hc2cf_16.o)
      ...
  "_fftw_rdft_hf_genus", referenced from:
      _desc in librdft_scalar_r2cf.a(hf_2.o)
      _desc in librdft_scalar_r2cf.a(hf_3.o)
      _desc in librdft_scalar_r2cf.a(hf_4.o)
      _desc in librdft_scalar_r2cf.a(hf_5.o)
      _desc in librdft_scalar_r2cf.a(hf_6.o)
      _desc in librdft_scalar_r2cf.a(hf_7.o)
      _desc in librdft_scalar_r2cf.a(hf_8.o)
      ...
  "_fftw_rdft_r2cbIII_genus", referenced from:
      _desc in librdft_scalar_r2cb.a(r2cbIII_2.o)
      _desc in librdft_scalar_r2cb.a(r2cbIII_3.o)
      _desc in librdft_scalar_r2cb.a(r2cbIII_4.o)
      _desc in librdft_scalar_r2cb.a(r2cbIII_5.o)
      _desc in librdft_scalar_r2cb.a(r2cbIII_6.o)
      _desc in librdft_scalar_r2cb.a(r2cbIII_7.o)
      _desc in librdft_scalar_r2cb.a(r2cbIII_8.o)
      ...
  "_fftw_rdft_r2cb_genus", referenced from:
      _desc in librdft_scalar_r2cb.a(r2cb_2.o)
      _desc in librdft_scalar_r2cb.a(r2cb_3.o)
      _desc in librdft_scalar_r2cb.a(r2cb_4.o)
      _desc in librdft_scalar_r2cb.a(r2cb_5.o)
      _desc in librdft_scalar_r2cb.a(r2cb_6.o)
      _desc in librdft_scalar_r2cb.a(r2cb_7.o)
      _desc in librdft_scalar_r2cb.a(r2cb_8.o)
      ...
  "_fftw_rdft_r2cfII_genus", referenced from:
      _desc in librdft_scalar_r2cf.a(r2cfII_2.o)
      _desc in librdft_scalar_r2cf.a(r2cfII_3.o)
      _desc in librdft_scalar_r2cf.a(r2cfII_4.o)
      _desc in librdft_scalar_r2cf.a(r2cfII_5.o)
      _desc in librdft_scalar_r2cf.a(r2cfII_6.o)
      _desc in librdft_scalar_r2cf.a(r2cfII_7.o)
      _desc in librdft_scalar_r2cf.a(r2cfII_8.o)
      ...
  "_fftw_rdft_r2cf_genus", referenced from:
      _desc in librdft_scalar_r2cf.a(r2cf_2.o)
      _desc in librdft_scalar_r2cf.a(r2cf_3.o)
      _desc in librdft_scalar_r2cf.a(r2cf_4.o)
      _desc in librdft_scalar_r2cf.a(r2cf_5.o)
      _desc in librdft_scalar_r2cf.a(r2cf_6.o)
      _desc in librdft_scalar_r2cf.a(r2cf_7.o)
      _desc in librdft_scalar_r2cf.a(r2cf_8.o)
      ...
  "_fftw_rdft_r2r_genus", referenced from:
      _desc in librdft_scalar_r2r.a(e01_8.o)
      _desc in librdft_scalar_r2r.a(e10_8.o)
  "_fftw_taint", referenced from:
      _mkplan in libdft.a(buffered.o)
      _mkplan in libdft.a(indirect-transpose.o)
      _mkplan in libdft.a(vrank-geq1.o)
      _mkplan in librdft.a(buffered.o)
      _mkcldw in librdft.a(hc2hc-direct.o)
      _mkplan in librdft.a(vrank-geq1.o)
      _mkcldrn_gcd in librdft.a(vrank3-transpose.o)
      ...
ld: symbol(s) not found for architecture x86_64
clang-6.0: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [Makefile:641: libfftw3.la] Error 1
make[1]: Leaving directory '/workspace/srcdir/fftw-3.3.9-alpha1/double'
make: *** [Makefile:707: install-recursive] Error 1
ERROR: LoadError: Build for FFTW on x86_64-apple-darwin14 did not complete successfully
```

--- a/build_tarballs.jl
+++ b/build_tarballs.jl
@@ -23,9 +23,15 @@ if [[ $target == x86_64-*  ]] || [[ $target == i686-* ]]; then config="$config -
 if [[ $target == *-w64-* ]]; then config="$config --with-our-malloc"; fi
 if [[ $target == i686-w64-* ]]; then config="$config --with-incoming-stack-boundary=2"; fi
 
+# work around GNU binutils problem on MacOS (see PR #1)
+if [[ ${target} == *darwin* ]]; then
+    export RANLIB=llvm-ranlib
+fi
+
 mkdir double && cd double
 ../configure $config
 perl -pi -e "s/tools m4/m4/" Makefile # work around FFTW/fftw3#146
+make -j${nprocs}
 make install
 cd ..
 
@@ -34,6 +40,7 @@ if [[ $target == arm-*  ]]; then config="$config --enable-neon"; fi
 mkdir single && cd single
 ../configure $config --enable-single
 perl -pi -e "s/tools m4/m4/" Makefile # work around FFTW/fftw3#146
+make -j${nprocs}
 make install
 """
 

